Tiltfile

```diff
@@ -18,9 +18,9 @@ settings = {
     "deploy_cert_manager": True,
     "preload_images_for_kind": True,
     "kind_cluster_name": "capg",
-    "capi_version": "v1.6.2",
+    "capi_version": "v1.7.0",
     "cert_manager_version": "v1.11.0",
```

Comment:
We could probably update the cert-manager to be in-line with the version automatically installed via clusterctl. But that can be a different PR.

Reply:
Will open a separate one for this.

Reply:
Done here: https://github.com/kubernetes-sigs/cluster-api-provider-gcp/pull/1210